app/services/importers/npq_contracts.rb

```diff
@@ -18,7 +19,7 @@ def call
         cohort = Cohort.find_by!(start_year: row["cohort_year"])
         cpd_lead_provider = CpdLeadProvider.find_by!(name: row["provider_name"])
         npq_lead_provider = cpd_lead_provider.npq_lead_provider
-        course = NPQCourse.where("name ilike '%#{row['course_name']}%'").first
+        course = NPQCourse.find_by!(identifier: row["course_identifier"])
 
         contract = NPQContract.find_or_initialize_by(
           cohort:,
```

Comment:
I think we need to be careful with versions, and either fail or bump up the version if there is an existing contract rather than attempt to update it 🤔 ?

Reply:
@cwrw 
I don't think we use versions with NPQContract, versions are used for ECF